liquibase-couchbase/src/test/java/integration/statement/RemoveDocumentsSqlPlusPlusQueryStatementIT.java

```diff
@@ -0,0 +1,113 @@
+package integration.statement;
+
+import com.couchbase.client.java.Collection;
+import com.couchbase.client.java.json.JsonObject;
+import com.couchbase.client.java.manager.query.CreatePrimaryQueryIndexOptions;
+import com.google.common.collect.Sets;
+import common.TransactionStatementTest;
+import common.operators.TestCollectionOperator;
+import liquibase.ext.couchbase.statement.RemoveDocumentsSqlPlusPlusQueryStatement;
+import liquibase.ext.couchbase.types.Document;
+import liquibase.ext.couchbase.types.Id;
+import liquibase.ext.couchbase.types.Keyspace;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import static common.constants.TestConstants.INDEX;
+import static common.matchers.CouchbaseCollectionAssert.assertThat;
+import static java.lang.String.format;
+import static liquibase.ext.couchbase.types.Keyspace.keyspace;
+
+class RemoveDocumentsSqlPlusPlusQueryStatementIT extends TransactionStatementTest {
+
+    private static final String DOC_FIELD_NAME = "field";
+    private static final String DOC_FIELD_VALUE = "val";
+    private static final String testCollection = UUID.randomUUID().toString();
+    private static final String testScope = UUID.randomUUID().toString();
```

Comment:
You could use `bucketOperator.createTestScope()` and `bucketOperator.createTestCollection(scopeName)` to generate test scope and collection. `bucketOperator()` already exists in `RandomizedScopeTestCase` which is the parent of this test